Recrutify/Recrutify.Services/DTOs/ProjectResultDTO.cs

```diff
@@ -11,6 +11,8 @@ public class ProjectResultDTO
 
         public StatusDTO Status { get; set; }
 
+        public bool IsAssign { get; set; }
```

Comment:
IsAssigned

Reply:
sorry, I did it right in the model but forgot to change it into DTO